docs/reliability/intro.md

```diff
@@ -11,13 +11,14 @@ To a certain extent, most
 of the previous techniques covered have to do with improving completion
 accuracy, and thus reliability, in particular self-consistency(@wang2022selfconsistency).
 However, there are a number of other techniques that can be used to improve reliability,
-beyond basic prompting strategies.
+beyond basic prompting strategies. 
 
-%%LLMs|LLM%% exhibit various problems including hallucinations(@ye2022unreliability), 
+While %%LLMs|LLM%% display a high level of charitable reliability even in the face of actively misleading prompts(@webson2023itscomplicated), 
```

Comment:
What is charitable reliability?

Reply:
LLMs reliably giving a prompt a charitable interpretation - essentially, the model can work its way around misguiding or incorrect data (e.g. misspelled words or bad grammar, or even fully tangential information in the same prompt) and understand what you're *trying* to say.

Reply:
Ah, can you define this here? People will not know it.

Reply:
Yep, sure thing!